students/JeffBennett/Lesson03/mailroom.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Data
+
+donors = [
+    ['Lorenzo Currie', 3000, 250, 5000],
+    ['Marie Kane', 1000, 195, 50.50],
+    ['John Beresford Tipton', 10000, 3000, 2500],
+    ['Janna Adams', 35.50, 19.75],
+    ['LeSean Haskell', 100],
+    ['Laila Samir', 1000, 250]
+]
+
+# Input/Output
+
+
+def print_menu():
+    """print menu of user choices."""
+    print('\nMain Menu\n' + '-'*25 +
+          '\n(1)\tSend a Thank You\n(2)\tCreate a Report\n(3)\tQuit')
+
+
+def print_donor_list(donors):
+    """print list of donor names."""
+    print('Our Current List of Donors:\n' + '-' * 25)
+    for item in donors:
+        print(item[0])
+
+
+def print_email(donor, gift):
+    """print email thanking donor for recent donation."""
+    print(f'\nDear {donor}\n')
+    print('We are delighted to acknowledge your generous')
+    print(f'gift of ${gift:,.2f} to Northwest Lifeboats.')
+    print('Your gift supports fishermen all along the Washington coast.\n')
+    print('Sincerely yours\n\nJ. A. Bennett\nDirector')
+
+
+def print_sorted_donors(sorted_donors):
+    """print list of donors sorted by total donation history."""
+    print('{0:32}{1:13}{2:14}{3:13}'.format
+          ('Donor Name', '| Total Gifts', '| Num Gifts', '| Average Gift'))
+    print('-' * 73)
+    for row in sorted_donors:
+        print('{0:30s}  ${1:12.2f}{2:12d}  ${3:12.2f}'.format(*row))
+
+
+# Processing
+
+
+def update_donors(list, name, donation):
```

Comment:
The `list` parameter is not used by this function. But you should also avoid using the names of Python built ins like `list` because this will overwrite them